doc/operator-scope.md

```diff
@@ -75,17 +83,48 @@ With the above changes the specified manifests should look as follows:
       name: memcached-operator
       apiGroup: rbac.authorization.k8s.io
     ```
-* `deploy/crds/cache.example.com_memcacheds_crd.yaml`
+
+## CRD scope
+
+Additionally the CustomResourceDefinition (CRD) scope can also be changed for cluster-scoped operators so that there is only a single instance (for a given name) of the CRD to manage across the cluster.
+
+> **NOTE**: Cluster-scoped CRDs are **NOT** supported with the Helm operator. While Helm releases can create cluster-scoped resources, Helm's design requires the release itself to be created in a specific namespace. Since the Helm operator uses a 1-to-1 mapping between a CR and a Helm release, Helm's namespace-scoped release requirement extends to Helm operator's namespace-scoped CR requirement.
+
+For each CRD that needs to be cluster-scoped, update its manifest to be cluster-scoped.
+
+* `deploy/crds/<full group>_<resource>_crd.yaml`
+  * Set `spec.scope: Cluster`
+
+To ensure that the CRD is always generated with `scope: Cluster`, add the tag `// +kubebuilder:resource:path=<resource>,scope=Cluster`, or if already present replace `scope={Namespaced -> Cluster}`, above the CRD's Go type definition in `pkg/apis/<group>/<version>/<kind>_types.go`. Note that the `<resource>` element must be the same lower-case plural value of the CRD's Kind, `spec.names.plural`. 
+
+### CRD cluster-scoped usage 
+
+It is ideal for the cases where an instance(CR) of some Kind(CRD) will be used in more than one namespace instead of an specific one. 
```

Comment:
```suggestion
This scope is ideal for the cases where an instance(CR) of some Kind(CRD) will be used in more than one namespace instead of a specific one. 
```